### ISSUE TITLE
prefer-default-export False Positive

### ISSUE THREAD

**Issue by j-a-y-h** [benmosher/eslint-plugin-import#1455]
**Description**
`eslint-plugin-import` v2.18.2 throws an error (`error  Prefer default export  import/prefer-default-export`) when there are multiple export statements in a typescript file. (this is probably not limited to typescript)

**Expected Results**
`eslint-plugin-import` v2.18.1 does not throw this error. This behavior aligns with the example page: [prefer-default-export](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md)

**Basic Example:**
```
export interface SubmitHandler {}
export function anything() {}
// error  Prefer default export  import/prefer-default-export
```

**Comment by xwatkins** [comment]
Experiencing the same issue with `2.18.1` and above but not with `2.18.0`.

**Comment by soullivaneuh** [comment]
I have the issue with the sapper template. Here is a code sample:

```
import posts from './_posts';

const lookup = new Map();
posts.forEach((post) => {
  lookup.set(post.slug, JSON.stringify(post));
});

export function get(req, res) {
  const { slug } = req.params;

  if (lookup.has(slug)) {
    res.writeHead(200, {
      'Content-Type': 'application/json',
    });

    res.end(lookup.get(slug));
  } else {
    res.writeHead(404, {
      'Content-Type': 'application/json',
    });

    res.end(JSON.stringify({
      message: 'Not found',
    }));
  }
}
```

Is it related to this issue?

**Comment by ljharb** [comment]
@soullivaneuh in that case, you only have a single named export - so the rule is correctly warning, telling you to make it a default export.

**Comment by soullivaneuh** [comment]
So I just have to do something like this?

```javascript
export.default = function get(req, res) {
// ...
```

I don't see any correct match on you examples,

I may open an another issue for that if you want.

**Comment by ljharb** [comment]
No, not like that, like this:
```js
export default function get() {}
```
